packages/help-center/src/components/help-center-footer.tsx

```diff
@@ -13,6 +16,10 @@ export const HelpCenterContactButton = () => {
 	const { sectionName } = useHelpCenterContext();
 	const redirectToWpcom = url === 'https://wordpress.com/help/contact';
 	const navigate = useNavigate();
+	const searchQuery = useSelect(
+		( select ) => ( select( HELP_CENTER_STORE ) as HelpCenterSelect ).getMessage(),
+		[]
+	);
```

Comment:
Can't we use this value directly in the search page instead?

Reply:
<img width="185" height="143" alt="Image" src="https://github.com/user-attachments/assets/3c348c8e-c5a1-4646-b20b-56d21ae376b9" />

Yes, we can

Reply:
Done! @alshakero